### ISSUE TITLE
Update install instructions

### ISSUE THREAD

**Issue by ocefpaf** [ioos/ioos_code_lab#54]
I'm going through the installation process again with @kbailey-noaa and we identified a few points to improve and/or fix:

- [X] We really need a redirect on all notebook_demos page! Just got bamboozled by that myself when googling for the installation;
- [X] Use Miniforge instead of Miniconda: https://github.com/conda-forge/miniforge/releases/tag/4.10.3-10 b/c that will simplify the channel configuration part and it will be on the safe side of the Anaconda Terms of conditions;
- [X] Make sure curl is installed on Windows/base env so we can simplify all the instructions to copy-n-paste code from the site;
- [X] Add a "remove-all-and-reinstall" note when facing update problems.

**Comment by MathewBiddle** [comment]
can we add an "if all else fails install from the lock file" section too?

**Comment by dpsnowden** [comment]
@MathewBiddle and @ocefpaf I just stumbled across this error in installing mamba forge and creating the environment.  I was copy/pasting from the [Create the IOOS conda environment](https://ioos.github.io/ioos_code_lab/content/ioos_installation_conda.html#create-the-ioos-conda-environment)

```
(base) ~ % mamba env create --quiet --file environment.yml


Encountered problems while solving:
  - nothing provides requested nco
  - nothing provides requested phantomjs
```

I found this issue on the mamba forge repo and it seems to have gone unaddressed.   https://github.com/mamba-org/mamba/issues/1145

For now I just deleted those two lines from the environment.yml file and it worked.  Mamba was able to create the environment and I was able to activate these.
